Source/CharacteristicNotificationManager.swift

```diff
@@ -63,6 +63,7 @@ class CharacteristicNotificationManager {
     }
 
     private func setNotifyValue(_ enabled: Bool, for characteristic: Characteristic) {
+        guard peripheral.state == .connected else { return }
```

Comment:
Please add logs to inform when this condition is not passed.
```suggestion
        guard peripheral.state == .connected else {
                RxBluetoothKitLog.w("""
                        \(peripheral.logDescription) is not connected. Changing notification state for not connected peripheral is not possible.
                        """)
                return 
        }
```

Reply:
Done, I splitted log message into two lines, because suggested change caused swiftlint warning.